Userland/Libraries/LibGfx/Alignment.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2022, Filiph Sandström <filiph.sandstrom@filfatstudios.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/StringView.h>
+#include <LibGfx/TextAlignment.h>
+
+namespace Gfx {
+
+#define GFX_ENUMERATE_ALIGNMENTS(M) \
+    M(Center)                       \
+    M(Left)                         \
+    M(Right)                        \
+    M(Bottom)                       \
+    M(Top)
+
+enum class Alignment {
```

Comment:
Is there a reason that we need yet another alignment property? Can't we just rename `TextAlignment` to `Alignment` and use it that way? This is what Qt does for the record.

Reply:
There's a bunch of places that use `TextAlignment`. Can we leave this up to another PR?

Reply:
I mean... Letting the IDE do a global rename and putting that in a commit doesn't seem too complicated...?

Reply:
@linusg sure